src/EnrollWebauthnController.js

```diff
@@ -88,9 +93,14 @@ function (Okta, Errors, FormType, FormController, CryptoUtil, webauthn, Footer,
               })
               .fail(function (error) {
                 self.trigger('errors:clear');
-                throw new Errors.WebAuthnError({
-                  xhr: {responseJSON: {errorSummary: error.message}}
-                });
+                // Do not display if it is abort error triggered by code when switching.
+                if (error && error.code === 20) {
+                  throw new Errors.WebauthnAbortError();
```

Comment:
its not really an error, is it? Can we just not do anything here? Or in other words, just handle this error when the code is not 20?

Reply:
Since we use BaseLoginModel here there is a fail case in base login model and it expects some know error else it fails. 

we already have similar cases 
https://github.com/okta/okta-signin-widget/blob/master/src/models/BaseLoginModel.js#L37